src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -78,6 +81,17 @@ public Navigation clickBackButton() {
     return this;
   }
 
+  public Navigation clickExploreWikiHeader() {
+    Loading loading = new Loading(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "loading" which hides the field declared at line 58. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)